### PR TITLE
Update about_Comparison_Operators.md

### PR DESCRIPTION
**Pull request by doctordns**
A large number of corrections to improve the MarkDown on this topic.

Joel noticed this and I saw it was just bad MD.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201701152229292882-953%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md [hunk 1]
```diff
@@ -28,20 +28,20 @@ separates these values.
...
--NotContains
--In
--NotIn
+-eq  
+-ne  
+-gt  
+-ge  
+-lt  
+-le  
+-Like
```
Comment (HemantMahawar) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md]: I believe these should start with lower-case, such as -like, -match etc.

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md [hunk 2]
```diff
@@ -28,20 +28,20 @@ separates these values.
```
Comment (HemantMahawar) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md]: Same set of changes need to be made in 3.0, 4.0 and 5.0 content

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md [hunk 3]
```diff
@@ -61,10 +61,11 @@ which always return a Boolean value.
 
 Windows PowerShell supports the following comparison operators.
 
--eq
-Description: Equal to. Includes an identical value.
-Example:
+-eq
```
Comment (HemantMahawar) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md]: This should be section header (for example see **Catch** in about_Language_Keywords.md for example)

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md [hunk 4]
```diff
@@ -73,11 +74,13 @@ False
 
 PS C:> "abc", "def" -eq "abc"
 abc
+```
 
--ne
-Description: Not equal to. Includes a different value.
-Example:
+-ne
```
Comment (HemantMahawar) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md]: Same comment about section header. More instances below

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md (hunk 1 above)
Comment (doctordns) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md]: fixed in next PR!

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C17%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201701171353347861-953%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C31%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201701310214053747-953%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by HemantMahawar** [comment]
Sorry @doctordns I missed that you addressed my comments with a new PR. Looks good ...

**Pull request merged**